pycon-za-2016/videos/closing.json

```diff
@@ -0,0 +1,23 @@
+{
+  "description": "<section class=\"section1\"><h1>Closing Ceremony</h1>\n<p>The closing ceremony. A general wrap of the conference. There will also be announcements about sprints on Saturday and Sunday.</p>\n</section>",
```

Comment:
Description field have to be in restructured text.
This is literally what you will see if published now:
> Description
  ~~~
  <section class="section1"><h1>Closing Ceremony</h1> <p>The closing ceremony. 
  A general wrap of the conference. There will also be announcements about sprints 
  on Saturday and Sunday.</p> </section>
  ~~~